webExtensionApis/study/src/studyUtils.js

```diff
@@ -704,6 +542,7 @@ function createLog(name, levelWord) {
   return L;
 }
 
+// TODO deal with these. Not needed, probably
```

Comment:
They are out of date. Web extensions have different signals, and we don't even utilize many events that sends "reason".